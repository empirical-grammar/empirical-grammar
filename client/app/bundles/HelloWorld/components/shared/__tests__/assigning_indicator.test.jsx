import React from 'react';
import { shallow } from 'enzyme';

import AssigningIndicator from '../button_loading_indicator';

describe('AssigningIndicator component', () => {

  it('should render', () => {
    const wrapper = shallow(
      <AssigningIndicator />
    );
    expect(wrapper).toMatchSnapshot();
  });

});
