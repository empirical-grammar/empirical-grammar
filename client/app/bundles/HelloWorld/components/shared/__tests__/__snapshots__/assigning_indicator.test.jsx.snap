// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssigningIndicator component should render 1`] = `
<span
  className="assigner-container"
>
  <img
    className="assigner"
    src="/images/assigner_still.png"
  />
</span>
`;
